api/axis.py

```diff
@@ -244,12 +244,12 @@ def __init__(self, _db, name="Axis 360 Circulation Monitor",
         else:
             # This should only happen during a test.
             self.metadata_wrangler = None
+        self.api = api or Axis360API.from_environment(self._db)
         self.bibliographic_coverage_provider = (
-            Axis360BibliographicCoverageProvider(self._db)
```

Comment:
You can't just call these constructors anymore, you have to pass in a specific API objet.